### PR TITLE
Tab improvements

### PR DESCRIPTION
**Pull request by brianegan**
Hello! Thanks so much for the sweet Hyperterm theme. Love the concept :)

One thing I figured could use a bit of polish: the tabs! Not sure if this is something you're interested in, but spent a little time playing with the tab styles for fun :) If you like the improvement, please feel free to merge / ask for updates!

Before:
screenshot 2016-07-31 13 57 59

After:
screenshot 2016-07-31 13 57 13

Before:

screenshot 2016-07-31 13 57 50

After:

screenshot 2016-07-31 13 57 07

### CODE REVIEW

**Comment by staltz** [comment]
Thanks! This looks like a good addition. I'll add a few comments inline

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 > Cyberpunk HyperTerm theme
 
-![](https://cdn.rawgit.com/staltz/hyperpunk/master/screenshot.png)
+![](screenshot.png)
```
Comment (staltz) [inline README.md:5]: We need to keep it how it was before so the screenshot shows also in the npm page.

Comment (brianegan) [inline README.md:5]: I think we should be safe here... if you check out another theme I made:
- https://github.com/brianegan/hyperterm-finest-base16-ocean-dark 

And the corresponding npm page for it: 
- https://www.npmjs.com/package/hyperterm-finest-base16-ocean-dark

The screenshot is showing up as expected. Does that work from your side?

Comment (staltz) [inline README.md:5]: Alright, let's see. :)

**Comment by staltz** [comment]
Version 1.1.0 released with this. Thanks!

**Comment by brianegan** [comment]
Awesome, thanks again for the sweet theme :)

File: README.md (hunk 1 above)
Comment (brianegan) [inline README.md:5]: Yay, it worked! https://www.npmjs.com/package/hyperpunk

**Pull request merged**